model/trigger/task.go

```diff
@@ -142,13 +145,21 @@ func (t *taskTriggers) Fetch(e *event.EventLogEntry) error {
 		return errors.New("couldn't find task")
 	}
 
+	t.version, err = version.FindOne(version.ById(t.task.Version))
+	if err != nil {
+		return errors.Wrap(err, "failed to fetch version")
+	}
+	if t.version == nil {
+		return errors.New("couldn't find version")
+	}
+
 	t.event = e
 
 	return nil
 }
 
 func (t *taskTriggers) Selectors() []event.Selector {
-	return []event.Selector{
+	selectors := []event.Selector{
 		{
```

Comment:
You'll need to add owner to this list of selectors

Reply:
It's added at the end